controllers/games-controllers.js

```diff
@@ -110,3 +111,13 @@ exports.getUsers = (req, res, next) => {
         res.status(200).send({ users: users });
     });
 };
+
+exports.deleteCommentsById = (req, res, next) => {
+    const { comment_id } = req.params;
+
+    removeComment(comment_id)
+        .then(() => {
+            res.status(204).send({});
```

Comment:
You don't need anything inside the brackets of the send, just send() is fine.
res.sendStatus(204) by itself could also be used here, as you don't need any response body whatsoever